src/components/Viewport.ts

```diff
@@ -674,7 +689,7 @@ export default class Viewport {
     }
 
     this.axes.destroy();
-    this.panInput.destroy();
+    this.panInput && this.panInput.destroy();
```

Comment:
nit
```diff
- this.panInput && this.panInput.destroy();
+ this.panInput!.destroy();
```

Reply:
I thought Flicking was still using Typescript v2.8, I'll fix that.